src/redis/cache_impl.go

```diff
@@ -154,6 +141,8 @@ func (this *rateLimitCacheImpl) DoLimit(
 	}
 
 	isOverLimitWithLocalCache := make([]bool, len(request.Descriptors))
+	results := make([]uint32, len(request.Descriptors), len(request.Descriptors))
```

Comment:
Is it intended? can you let us know why we need to specify `make([]uint32, len(request.Descriptors), len(request.Descriptors))` instead of `make([]uint32, len(request.Descriptors))`

Reply:
No, it's not intended. Will remove the redundant.